index.d.ts

```diff
@@ -19,6 +19,14 @@ export interface Options {
    * @default process.cwd()
    */
   repo?: string
+
+  /**
+   * Path(s) to a file or directory to limit results to specific changes. When no
+   * `path` is specified, the entire history of the project id returned.
+   *
+   * @default ''
```

Comment:
I think it's more clear to not have a default value here, since not passing `path` and passing `path: ''` is going to produce different behaviour?

```suggestion
```